clash-protocols-memmap/src/Protocols/MemoryMap/Check.hs

```diff
@@ -0,0 +1,12 @@
+-- SPDX-FileCopyrightText: 2025 Google LLC
+--
+-- SPDX-License-Identifier: Apache-2.0
+
+-- | Validity checks performed on memory maps
+module Protocols.MemoryMap.Check (
+  module Protocols.MemoryMap.Check.AbsAddress,
+  module Protocols.MemoryMap.Check.Normalised,
```

Comment:
I'm going to be annoying and say that we decided to do everything in US English a couple of years ago, sorry :(